core/spring-cloud-stream/src/main/java/org/springframework/cloud/stream/endpoint/BindingsEndpoint.java

```diff
@@ -53,7 +53,7 @@ public List<?> queryStates() {
 	}
 
 	@ReadOperation
-	public Binding<?> queryState(@Selector String name) {
+	public List<Binding<?>> queryState(@Selector String name) {
```

Comment:
A noteworthy change is that the queryState operation will now return a list instead of a single element, which modifies the json generated by the response.

Reply:
That makes sence